components/expenses/ExpenseFormPayeeStep.js

```diff
@@ -136,6 +136,12 @@ const ExpenseFormPayeeStep = ({
     __background__: 'white',
   });
 
+  useEffect(() => {
+    setLocationFromPayee(formik, collective.host);
+
+    return () => setLocationFromPayee(formik, collective.host);
```

Comment:
You don't need to return an unmount callback here.
```suggestion
    return () => ();
```